spec/requests/home_controller_spec.rb

```diff
@@ -0,0 +1,20 @@
+require 'rails_helper'
+
+RSpec.describe 'HomeController', type: :request do
+  describe 'GET /index' do
+    it 'should render the correct response status' do
+      get root_path
+      expect(response).to have_http_status(:ok)
+    end
+
+    it 'should render the correct Template' do
+      get root_path
+      expect(response).to render_template(:index)
+    end
+
+    it 'renders the correct placeholder text of the body' do
+      get root_path
+      expect(response.body).to include('Welcome to Encarta Blog App!')
+    end
+  end
+end
```

Comment:
- [ ] **[OPTIONAL]** I don't think we need a home controller. The index action in the users controller should serve as the root path. You can update the route file and set the root path from

  ```
  home#index
  ```
  to
  ```
  users#index
  ```

Reply:
Thanks for the suggestion. I will put that into implementation.